packages/editor-ui/src/components/ValueSurvey.vue

```diff
@@ -0,0 +1,189 @@
+import { workflowHelpers } from "@/components/mixins/workflowHelpers";
+import ModalDrawer from './ModalDrawer.vue';
+
+const DEFAULT_TITLE = `How would you feel if you could <strong>no longer use n8n</strong>?`;
+const VERY_TITLE = `Great to hear! Can we reach out to see how we can make n8n even better for you?`;
+const SOMEWHAT_TITLE = `Thanks for your feedback! We'd love to understand how we can improve. Can we reach out?`;
+const NOT_TITLE = `Sorry to hear that. We'd love to learn how to improve. Can we reach out?`;
+
+export default mixins(workflowHelpers).extend({
+	name: 'ValueSurvey',
+	components: {
+		ModalDrawer,
+	},
+	computed: {
+		getTitle (): string {
+			if (this.form.value === 'very') {
+				return VERY_TITLE;
+			} else if (this.form.value === 'somewhat') {
+				return SOMEWHAT_TITLE;
+			} else if (this.form.value === 'not') {
+				return NOT_TITLE;
+			} else {
+				return DEFAULT_TITLE;
+			}
+		},
+		isEmailValid(): boolean {
+			return VALID_EMAIL_REGEX.test(String(this.form.email).toLowerCase());
+		},
+	},
+	data() {
+		return {
+			form: {
+				email: '',
+				value: '',
+			},
+			showButtons: true,
+			VALUE_SURVEY_MODAL_KEY,
+		};
+	},
+	methods: {
+		closeDialog(): void {
+			if (this.form.value === '') {
+				this.$telemetry.track('User responded value survey score', { instance_id: this.$store.getters.instanceId, how_disappointed: '' });
+			} else if (this.form.value !== '') {
+				this.$telemetry.track('User responded value survey email', { instance_id: this.$store.getters.instanceId, email: '' });
+			}
+
+			this.$store.commit('ui/closeTopModal');
+		},
+		selectSurveyValue(value: string) {
+			this.form.value = value;
+			this.showButtons = false;
+			this.$store.dispatch('settings/submitValueSurvey', {value: this.form.value});
+			this.$telemetry.track('User responded value survey score', { instance_id: this.$store.getters.instanceId, how_disappointed: this.form.value });
+		},
+		send(): void {
+			if (this.isEmailValid) {
+				this.$store.dispatch('settings/submitValueSurvey', {email: this.form.email, value: this.form.value});
+				this.$telemetry.track('User responded value survey email', { instance_id: this.$store.getters.instanceId, email: this.form.email });
+				this.$showMessage({
+					title: 'Thanks for your feedback',
+					message: `If you’d like to help even more, answer this <a target="_blank" href="https://n8n-community.typeform.com/quicksurvey#how_disappointed=${this.form.value}&instance_id=${this.$store.getters.instanceId}">quick survey.</a>`,
+					type: 'success',
+					duration: 15000,
+				});
+				this.$store.commit('ui/closeTopModal');
+			}
+		},
+	},
+	mounted() {
+		this.$telemetry.track('User shown value survey', { instance_id: this.$store.getters.instanceId});
+	},
+});
+</script>
+
+<style module lang="scss">
+.title {
+	height: 16px;
+	font-size: var(--font-size-m);
+	line-height: 18px;
+	text-align: center;
+	color: var(--color-text-xlight);
```

Comment:
I think we can do this with heading component.. otherwise we should update it to support it. height can be applied to parent

Reply:
Fixed :)